yellowbrick/classifier/class_balance_heat_map.py

```diff
@@ -0,0 +1,169 @@
+# yellowbrick.classifier.class_balance_heat_map
```

Comment:
I think one thing we need to do is have a discussion of whether this is its own visualizer or if it should simply extend the existing `ClassBalance` visualizer. Either way, I don't believe this needs to be in a separate file, I recommend adding it to the `yellowbrick/classifier/class_balance.py` module. 

This makes things a bit easier for you as well since the documentation and test files are already set up!